nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc

```diff
@@ -12,6 +12,35 @@ These release notes track the development of the {descheduler-operator}.
 
 For more information, see xref:../../../nodes/scheduling/descheduler/index.adoc#nodes-descheduler-about_nodes-descheduler-about[About the descheduler].
 
+[id="descheduler-operator-release-notes-5.0.1"]
```

Comment:
```suggestion
[id="descheduler-operator-release-notes-5.0.1_{context}"]
```

Reply:
Yeah I had thought about doing this but didn't because the rest of them in the file didn't have them. If I do this I'll do it in a full sweep of the files in a separate PR sometime. Thanks!